scrunch/datasets.py

```diff
@@ -1249,7 +1249,8 @@ def add_filter(self, name, expr, public=False):
         new_filter = self.resource.filters.create(payload)
         return self.filters[new_filter.body['name']]
 
-    def create_single_response(self, categories, name, alias, description='', missing=True):
+    def create_single_response(self, categories, name, alias, description='',
+                               notes='', missing=True):
```

Comment:
New parameters should go to the end, if not any current clients using positional arguments might get broken behavior.